java/Dockerfile.tpl

```diff
@@ -48,6 +48,10 @@ RUN echo "Starting ..." && \
     apt-get -qq -y install graphviz && \
     echo "Done Install graphviz!" && \
 
+    echo "Install jq" && \
+    apt-get -qq -y install jq && \
```

Comment:
Should be done in the base install above?

Reply:
Yeah, I guess it's not "bug enough" to be considered at the same level thant graphviz or maven.
I'll move it :ok_hand: .
I'll still test it though just to keep things clean.

Reply:
Done